quickstarts/confluent-cloud/config.yml

```diff
@@ -0,0 +1,61 @@
+id: 
```

Comment:
We generate the `id` field through the merge process - We can get rid of this line

Reply:
Removed.